src/mod_mbedtls.inl

```diff
@@ -133,10 +221,43 @@ mbed_sslctx_uninit(SSL_CTX *ctx)
 	mbedtls_ctr_drbg_free(&ctx->ctr);
 	mbedtls_pk_free(&ctx->pkey);
 	mbedtls_x509_crt_free(&ctx->cert);
-	mbedtls_entropy_free(&ctx->entropy);
+	//mbedtls_entropy_free(&ctx->entropy);
 	mbedtls_ssl_config_free(&ctx->conf);
 }
 
+static int tls_tx(void *ctx, const unsigned char *buf, size_t len)
```

Comment:
Can't we use `tls_tx` from `sockets_tls`? Or at lease mirror it?

Reply:
I will look into it.